This fixes the following error on ppc64le:

    o2hbmonitor.c: In function 'get_device_name':
    o2hbmonitor.c:124:12: error: 'PATH_MAX' undeclared (first use in this function)
      char path[PATH_MAX];

--- a/o2monitor/o2hbmonitor.c
+++ b/o2monitor/o2hbmonitor.c
@@ -45,6 +45,7 @@
 #include <errno.h>
 #include <sys/ipc.h>
 #include <sys/sem.h>
+#include <limits.h>
 
 #define SYS_CONFIG_DIR			"/sys/kernel/config"
 #define O2HB_CLUSTER_DIR		SYS_CONFIG_DIR"/cluster"
